authors/apps/article/views.py

```diff
@@ -1,26 +1,21 @@
+from django.db.models import Avg
 from django.db.utils import IntegrityError
+from rest_framework import status
+from rest_framework.exceptions import APIException
+from rest_framework.exceptions import NotFound, ValidationError
+from rest_framework.generics import GenericAPIView, ListCreateAPIView, RetrieveAPIView
 from rest_framework.permissions import IsAuthenticated, IsAuthenticatedOrReadOnly
 from rest_framework.response import Response
-from rest_framework.exceptions import APIException, NotFound, ValidationError
 from rest_framework.views import APIView
-from rest_framework import serializers, status
-from django.db.models import Avg
 
-
-from .serializers import ArticleSerializer, CommentSerializer, DeleteCommentSerializer, RateSerializer, BookmarksSerializer
-from ..profiles.models import UserProfile
-from .models import Article, ArticleImage, ArticleLikes, Rate, ArticleFavourite, ArticleComment, LikeComment, ArticleBookmark, ReportArticle
-from rest_framework.generics import GenericAPIView , ListCreateAPIView, RetrieveAPIView
-from rest_framework.exceptions import NotFound, ValidationError
-from rest_framework import serializers, status
-from django.template.loader import render_to_string
-from django.utils.html import strip_tags
-from django.core.mail import EmailMultiAlternatives
-from ..profiles.models import UserProfile
-from .serializers import ArticleSerializer, CommentSerializer, DeleteCommentSerializer, RateSerializer,ReportSerializer
-from .renderer import ArticleJSONRenderer, CommentJSONRenderer
 from authors.apps.authentication.utils import send_email
+from .models import Article, ArticleImage, ArticleLikes, Rate, ArticleFavourite, ArticleComment, LikeComment, \
+    ArticleBookmark, ReportArticle
+from .renderer import ArticleJSONRenderer, CommentJSONRenderer
+from .serializers import ArticleSerializer, CommentSerializer, DeleteCommentSerializer, RateSerializer, ReportSerializer
+from .serializers import BookmarksSerializer
```

Comment:
I don't think there's need to have this as a separate import since it's from the same file as the other serializers

Reply:
Hey @JoanNgatia I did not make a separate import. My linter had reformatted the code. In this case it appeared like I had deleted and re-imported the same items

Reply:
Oh okay then, Please fix the merge conflicts and then we can merge